django/chat/templates/chat/components/chat_welcome.html

```diff
@@ -2,7 +2,7 @@
 
 <div id="no-messages-placeholder" class="d-flex flex-column">
   <div class="flex-grow-1 d-flex align-items-center justify-content-center">
-    <div>
+    <div id="chat-welcome">
```

Comment:
A number of small changes across the codebase to add IDs or container elements for improved highlighting during the tour.